js/views/modals/purchase/Moderators.js

```diff
@@ -184,7 +184,7 @@ export default class extends baseVw {
       this.removeNotFetched(model.id);
       this.modCards.push(newModView);
       // if required, select the first  moderator
-      if (this.options.selectFirst && !this.firstSelected) {
+      if (this.options.selectFirst && !this.firstSelected && !this.noneSelected) {
```

Comment:
noneSelected is set if no moderators should be selected.

This should be set to false if the total of the order is too low for moderation, since deselecting all the moderators won't affect moderators that aren't loaded yet.